### ISSUE TITLE
Error_fetch_data - Afer install no way to make it works, what is worng ?

### ISSUE THREAD

**Issue by gamersalpha** [ItzCrazyKns/Perplexica#454]
**Describe the bug**
When i start the docker i got this error : 

**To Reproduce**

just start the researche page, i can't do nothing

**Expected behavior**
works

**Screenshots**
https://i.imgur.com/L5Y3E5i.png

**Additional context**

**Comment by gamersalpha** [comment]
i can't send image here :/

**Comment by gamersalpha** [comment]
Hello i start evrithung i found the docker don't start with this error : 

https://gist.github.com/gamersalpha/e2ce04501e349e83ef8909f3ee619357


what is wrong please

**Comment by ItzCrazyKns** [comment]
Go the the directory where Perplexica was cloned, remove the folder named config.toml and rename the sample.config.toml to config.toml and then reconfigure it. After that, restart the container, your issue should be fixed.

**Comment by gamersalpha** [comment]
Hello works but i got a new problem : 

![image](https://github.com/user-attachments/assets/bc9c8cfe-af27-4d65-8d31-7a882cfd3a0a)


![image](https://github.com/user-attachments/assets/03c9c80b-5076-4c85-8722-421063255d24)





```
yarn run v1.22.19
$ npm run db:push && node dist/app.js
> perplexica-backend@1.9.2 db:push
> drizzle-kit push sqlite
drizzle-kit: v0.22.7
drizzle-orm: v0.31.2
No config path provided, using default path
Reading config file '/home/perplexica/drizzle.config.ts'
[⣷] Pulling schema from database...
[✓] Pulling schema from database...
[✓] Changes applied
npm notice
npm notice New minor version of npm available! 10.7.0 -> 10.9.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.9.0
npm notice To update run: npm install -g npm@10.9.0
npm notice
info: WebSocket server started on port 3001
info: Server is running on port 3001
error: Error in discover route: Request failed with status code 403
error: Error in discover route: Request failed with status code 403
error: Error in discover route: Request failed with status code 403
error: Error in discover route: Request failed with status code 403
error: Error in discover route: Request failed with status code 403
error: Error in discover route: Request failed with status code 403
error: Error in discover route: Request failed with status code 403
error: Unhandled Rejection at: [object Promise], reason: AxiosError: Request failed with status code 403
error: Error in image search: Request failed with status code 403
error: Unhandled Rejection at: [object Promise], reason: AxiosError: Request failed with status code 403
error: Error in discover route: Request failed with status code 403
error: Unhandled Rejection at: [object Promise], reason: AxiosError: Request failed with status code 403
error: Unhandled Rejection at: [object Promise], reason: AxiosError: Request failed with status code 403

```

**Comment by ItzCrazyKns** [comment]
You're using a public SearXNG instance, it is rate limited & probably doesn't has JSON support. Use the container I provide, the issue will be fixed.
